backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -76,9 +77,13 @@ def resolve_fetch_failures_0_members(
 
         # Update DB
         if success_cases:
-            with get_db_connection(schema=schema, local=use_local_db) as con:
-                concept_set_members__from_csets_and_members_to_db(con, csets_and_members)
-                refresh_derived_tables(con)
+            try:
+                with get_db_connection(schema=schema, local=use_local_db) as con:
+                    concept_set_members__from_csets_and_members_to_db(con, csets_and_members)
+                    refresh_derived_tables(con)
+            except Exception as err:
+                reset_temp_refresh_tables(schema)
```

Comment:
## Discussion
This is the update.

This is prompted by the fact that I saw this error message pop up in this [fetch failures log](https://github.com/jhu-bids/TermHub/actions/runs/8212911832/job/22463708956):
`sqlalchemy.exc.ProgrammingError: (psycopg2.errors.DuplicateTable) relation "all_csets_old" already exists`

This is unexpected; I haven't seen this error pop up in `resolve_fetch_failures` before. This update doesn't fix the problem. It only ensures a clean exit if the problem does occur. So the problem can occur again. But since it happens rarely, and since the fetch failures script will run again in a day anyway, I guess it's not a huge problem.

The primary cause that I can think of for this happening would be if this script and the normal refresh were running simultaneously. However, the way I've set up `refresh_derived_tables()`, only one of these things should run at a time. So maybe that logic is broken. Other than that, I have no idea why this would occur.